schemata/g6-iaas-schema.json

```diff
@@ -0,0 +1,1028 @@
+{
```

Comment:
Hmm, I'm not sure about schemata. While strictly correct in terms of Greek roots it's not the common pluralisation in technical uses. For example, database schemas. See: http://english.stackexchange.com/questions/77764/plural-form-of-schema